core/src/Streamly/Internal/FileSystem/WindowsPath.hs

```diff
@@ -1,2 +1,452 @@
 #define IS_WINDOWS
 #include "Streamly/Internal/FileSystem/PosixPath.hs"
+
+{- $setup
```

Comment:
I assumed that this should be on top of the module.
Do the doctests still work if the setup is after `#include "Streamly/Internal/FileSystem/PosixPath.hs"`?

Reply:
Yes. it has to be before the tests, that's all.